core/lcsb/rems.py

```diff
@@ -155,37 +189,19 @@ def handle_rems_entitlement(data: Dict[str, str]) -> bool:
     return True
 
 
-def create_rems_entitlement(obj: Union[Access, User], 
-        application: str, 
-        dataset_id: str, 
-        user_oidc_id: str, 
-        email: str,
-        expiration_date: date) -> bool:
+def create_rems_entitlement(obj: Union[Access, User],
```

Comment:
This looks like a black-linted line. Should it not be all parameters on a new line?